packages/next-app/src/app/metadata/[network]/pool/[poolId]/(components)/MetadataAttributesTable/index.tsx

```diff
@@ -219,6 +235,7 @@ export default function MetadataAttributesTable({
           poolId={poolId}
           isOwner={isOwner}
           metadataUpdated={metadataUpdated}
+          isMetadataValid={isMetadataValid}
```

Comment:
We could probably consume from this context directly in `Actions` right?

Reply:
Yes, we could. We can move both `metadataUpdated` and `isMetadataValid` to `Actions`